web/pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx

```diff
@@ -800,6 +801,7 @@ export function ResultSet() {
       setSelectedColumns(new Set());
       rsu.current.resetClientPKIndex();
       setLoaderText(gettext('Waiting for the query to complete...'));
+      setQuery(query);
```

Comment:
Use meaningful name. `setDataOutputQuery`/`setResultQuery`

Reply:
I have changed it to setDataOutputQuery